md2conf.py

```diff
@@ -136,7 +136,7 @@
     else:
         LOGGER.error('Error: Org Name not specified by environment variable or option.')
         sys.exit(1)
-
```

Comment:
An even better change might be to rename `CONFLUENCE_API_URL_TMP` back to `CONFLUENCE_API_URL`.